connectors/kafka/src/main/java/forklift/connectors/KafkaConnector.java

```diff
@@ -0,0 +1,167 @@
+package forklift.connectors;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import forklift.consumer.ForkliftConsumerI;
+import forklift.consumer.KafkaTopicConsumer;
+import forklift.message.KafkaMessage;
+import forklift.producers.ForkliftProducerI;
+import forklift.producers.KafkaForkliftProducer;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import javax.jms.Connection;
+import javax.jms.Message;
+
+/**
+ * Manages a {@link org.apache.kafka.clients.consumer.KafkaConsumer}.  A subscription is made whenever  a call to getQueue or get Topic
+ * is received.  Subscriptions are removed if no consumption is seen for
+ */
+public class KafkaConnector implements ForkliftConnectorI {
+    private static final Logger log = LoggerFactory.getLogger(KafkaConnector.class);
+    private final String kafkaHosts;
+    private final String schemaRegistries;
+    private final String groupId;
+    private KafkaConsumer<?, ?> kafkaConsumer;
+    private KafkaProducer<?, ?> kafkaProducer;
+    private MessageStream messageStream = new MessageStream();
+
+    static ObjectMapper mapper = new ObjectMapper().registerModule(new JavaTimeModule())
+                                                   .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    KafkaController controller;
```

Comment:
Yea I could prefer to see these be `protected` because of general confusion around the `default` modifier if the access modifier is just for testing